src/com/gfredericks/test/chuck/clojure_test.cljc

```diff
@@ -0,0 +1,98 @@
+(ns com.gfredericks.test.chuck.clojure-test
+  (:require [clojure.test.check :as tc]
+            [clojure.test.check.properties :as prop
+             #?@(:cljs [:include-macros true])]
+            #?(:clj  [clojure.test :as ct :refer [is]]
+               :cljs [cljs.test :as ct :refer-macros [is]])))
+
+(defn ^:private not-exception?
+  [value]
+  (not (instance? #?(:clj  Throwable
+                     :cljs js/Error)
+                  value)))
+
+(defn report-exception [result]
+  (is (not-exception? (:result result)) result))
+
+(defn pass? [reports]
+  (every? #(= (:type %) :pass) reports))
+
+(defn report-needed? [reports final-reports]
+  (or (not (pass? reports)) (empty? final-reports)))
+
+(defn save-to-final-reports [final-reports reports]
+  (if (report-needed? reports final-reports)
+    reports
+    final-reports))
+
+(def ^:dynamic *chuck-captured-reports*)
+
+#?(:cljs
+(defmethod ct/report [::chuck-capture :fail]
+  [m]
+  (swap! *chuck-captured-reports* conj m)))
+
+#?(:cljs
+(defmethod ct/report [::chuck-capture :pass]
+  [m]
+  (swap! *chuck-captured-reports* conj m)))
+
+(defn capture-reports*
+  [reports-atom f]
+  #?(:clj
+     (binding [ct/report #(swap! reports-atom conj %)]
+       (f))
+
+     :cljs
+     (binding [*chuck-captured-reports* reports-atom
+               cljs.test/*current-env* (cljs.test/empty-env ::chuck-capture)]
+       (f))))
+
+(defmacro capture-reports
+  [& body]
+  `(let [reports# (atom [])]
+     (capture-reports* reports# (fn [] ~@body))
+     @reports#))
+
+(defmacro qc-and-report-exception
+  [final-reports tests bindings & body]
+  `(report-exception
+    (tc/quick-check
+      ~tests
+      (prop/for-all ~bindings
+        (let [reports# (capture-reports ~@body)]
+          (swap! ~final-reports save-to-final-reports reports#)
+          (pass? reports#))))))
+
+(defn -testing
+  [name func]
+  (#?(:clj clojure.test/testing :cljs cljs.test/testing) name (func)))
```

Comment:
Well, of course you were right, and I was wrong. It worked with `:refer-macros [testing]`. It's done in 23f34e3.

I just checked and what I tried the other day was `[cljs.test :as ct :refer-macros [is testing]]` but then tried to call the macro as `(ct/testing ...)` instead of `(testing ...)`.